app/views/certificates.html

```diff
@@ -60,8 +60,8 @@
                   </td>
                 </tr>
                 <tr>
-                  <td width="120">Comman Name: </td>
```

Comment:
maybe have a toggle option to not show the input field by default. something like

<a data-toggle="collapse" data-target="#passphrase">Supply a passphrase</a>
<div class="collapse" id="passphrase">
input box to show on toggle
</div>